fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1072,6 +1077,14 @@ public ReplicationMgr getReplicationMgr() {
         return replicationMgr;
     }
 
+    public LockManager getLockManager() {
```

Comment:
How about using LockManager.getSingleton()? We have so many xxxMgr which should not put in GlobalState.